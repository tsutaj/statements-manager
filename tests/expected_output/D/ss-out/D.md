


## Problem D

### <u>**A+B**</u>

2 つの整数 $A$, $B$ が与えられます。$A + B$ を出力してください。

$T$ 個のテストケースが与えられますので、$T$ 行出力してください。

### Input

```
$T$
$A_1$ $B_1$
$A_2$ $B_2$
...
$A_T$ $B_T$
```

- 入力は全て整数で与えられる
- $1 \leq T \leq 100{,}000$
- $1 \leq A_i, B_i \leq 10^{9}$

### Output

$T$ 行出力してください。

$i$ 行目には $A_i + B_i$ の計算結果を出力してください。


### Sample Input 
```
3
3 9
100 101
1 10
```



### Sample Output 
```
12
201
11
```








