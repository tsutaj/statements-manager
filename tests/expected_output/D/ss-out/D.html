<!DOCTYPE html>
<html lang="ja">

<head>
    <title>Problem Statement</title>
    <meta charset="utf-8">
    <link href='http://fonts.googleapis.com/css?family=Noto+Serif' rel='stylesheet' type='text/css'>
    <style type="text/css">
        html {
            color: #222222;
            font-family: "Noto Serif";
        }

        h1 {
            font-size: 16pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
        }

        h2 {
            font-size: 14pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h3 {
            font-size: 14pt;
            border-bottom: solid 2px #e0e0e0;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h4 {
            font-size: 13pt;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        h5 {
            font-size: 12pt;
            margin-bottom: 20px;
            margin-top: 20px;
        }

        b {
            font-weight: bold;
        }

        p {
            font-size: 12pt;
        }

        li {
            font-size: 12pt;
        }

        pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #e0e0e0;
            border-radius: 4px;
        }

        code {
            color: #e83e8c;
        }

        code.language-input-format {
            color: #222222;
        }

        .testcase_pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #e0e0e0;
            border-radius: 4px;
            overflow: auto;
            max-width: 480;
            max-height: 340;
        }

        .error_pre {
            font-size: 11pt;
            padding: 8px;
            border: solid 1px #f0e0e0;
            border-radius: 4px;
            overflow: auto;
            max-height: 340;
            background-color: #ffeeee;
        }

        .table tbody>tr>td.vert-aligned {
            vertical-align: middle;
        }
    </style>
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
            tex2jax: {
                inlineMath: [["$", "$"]],
                processEscapes: true,
                processClass: 'input-format|language-input-format'
            }
        });
    </script>
    <script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js?config=TeX-AMS_CHTML">
        </script>
</head>

<body>
    
    <div >
        <h2>Problem D</h2>
<h3><u><strong>A+B</strong></u></h3>
<p>2 つの整数 $A$, $B$ が与えられます。$A + B$ を出力してください。</p>
<p>$T$ 個のテストケースが与えられますので、$T$ 行出力してください。</p>
<h3>Input</h3>
<pre class="input-format"><code class="language-input-format">$T$
$A_1$ $B_1$
$A_2$ $B_2$
...
$A_T$ $B_T$
</code></pre>
<ul>
<li>入力は全て整数で与えられる</li>
<li>$1 \leq T \leq 100{,}000$</li>
<li>$1 \leq A_i, B_i \leq 10^{9}$</li>
</ul>
<h3>Output</h3>
<p>$T$ 行出力してください。</p>
<p>$i$ 行目には $A_i + B_i$ の計算結果を出力してください。</p>
<h3>Sample Input</h3>
<pre class="input-format"><code class="language-input-format">3
3 9
100 101
1 10
</code></pre>
<h3>Sample Output</h3>
<pre class="input-format"><code class="language-input-format">12
201
11
</code></pre>
    </div>
    
</body>

</html>